unit-tests/run-unit-tests.py

```diff
@@ -56,7 +55,7 @@ def progress(*args):
     def out(*args):
         print( *args )
     def progress(*args):
-        print( *args )
+        print( *args, end='\r' )
```

Comment:
Why?

Reply:
so that there is always 1 line printed in the terminal. Same as it is when there is color support

Reply:
So with
```
line 1 is long
line 2
```
You'll get
```
line 2 is long
```
??

Reply:
yes.... but this is how it was before. I was just trying to keep it unchanged

Reply:
How it was before?

Reply:
Before we changed the script, when I started working

Reply:
I don't understand... I see this as something new. If the stream is not a terminal, we cannot do the `clear_eol + '\r'` that we do above... we cannot expect a progress line to overwrite the previous

Reply:
Ok, I'll remove it